ansys/mapdl/reader/rst.py

```diff
@@ -233,7 +233,16 @@ def _read_result_header(self):
         standard_header = read_standard_header(self.filename)
 
         # Read .RST FILE HEADER
-        header = parse_header(self.read_record(103), result_header_keys)
+        try:
+            record = self.read_record(103)
+            if record.size > 80:
+                raise RuntimeError("Invalid record size. Should be 80 entries")
+            header = parse_header(record, result_header_keys)
+        except:
+            raise RuntimeError(
```

Comment:
Also I rather you re-raise the initial error, just pre-pending the new message to the previous error message, so you do not hide potentially useful information.

Reply:
Agreed. Opted for checking the header size rather than placing this in a try/except.